package.json

```diff
@@ -35,6 +35,7 @@
 		"@semantic-release/changelog": "^5.0.1",
 		"@semantic-release/exec": "^5.0.0",
 		"@semantic-release/git": "^9.0.0",
+		"@wordpress/block-editor": "^4.3.7",
```

Comment:
Why did you add that?

Reply:
this was for the QR Code, didn't pay attention and I installed in the wrong place.